core/src/main/java/org/apache/iceberg/avro/AvroSchemaUtil.java

```diff
@@ -76,8 +76,10 @@ public static Schema convert(Type type, Map<Types.StructType, String> names) {
     return TypeUtil.visit(type, new TypeToSchema(names));
   }
 
-  public static Type convert(Schema schema) {
-    return AvroSchemaVisitor.visit(schema, new SchemaToType(schema));
+  public static org.apache.iceberg.Schema convert(Schema schema) {
+    final Type type = AvroSchemaVisitor.visit(schema, new SchemaToType(schema));
+    final List<Types.NestedField> fields = type.asNestedType().asStructType().fields();
+    return new org.apache.iceberg.Schema(fields);
```

Comment:
Can we do this without a breaking API change? What about adding a different name to convert directly to a Schema?

Reply:
Done!